docs/imviz/displayimages.rst

```diff
@@ -411,7 +411,7 @@ To set the percentile for all the images at once:
 Colormap
 --------
 
-The spectrum of colors used to visualize data can be changed using this drop down.
+The spectrum of colors used to visualize data can be changed using this drop-down.
```

Comment:
@rosteen had a beef on "drop down" vs "dropdown" vs "drop-down" but I don't remember if I ever agreed what to use in the end.

Reply:
My order of preference is dropdown, drop-down, then drop down 😂

Reply:
I think my conclusion after reading up on best practices/conventions was "you shouldn't use any version of 'dropdown', just call it a menu" so it doesn't really matter 😆